test/fail_compilation/test18708.d

```diff
@@ -0,0 +1,64 @@
+/* TEST_OUTPUT:
+---
+fail_compilation/test18708.d(24): Error: one path skips field `s`
```

Comment:
Can the missing path be included in the error message?

Reply:
Changing error messages is beyond the scope of this PR.